grasping_benchmarks/graspnet/graspnet_grasp_planner.py

```diff
@@ -68,16 +152,19 @@ def configure(self, cfg : dict):
 
         # Create a namespace from the config dict
         # Since the GraspNet implementation uses a namespace
-        self.cfg_ns = SimpleNamespace(**self.cfg)
 
-        self.grasp_sampler_args = utils.read_checkpoint_args(self.cfg_ns.grasp_sampler_folder)
-        self.grasp_sampler_args.is_train = False
-        self.grasp_evaluator_args = utils.read_checkpoint_args(self.cfg_ns.grasp_evaluator_folder)
-        self.grasp_evaluator_args.continue_train = True
 
-        self.estimator = grasp_estimator.GraspEstimator(self.grasp_sampler_args,
-                                                        self.grasp_evaluator_args,
-                                                        self.cfg_ns)
+        self.cfg_ns = SimpleNamespace(**self.cfg)
+        print('you are here')
+        self.cfg_grasp_estimator = grasp_estimator.joint_config(
+            self.cfg_ns.vae_checkpoint_folder,
+            self.cfg_ns.evaluator_checkpoint_folder,
+        )      
+        self.cfg_grasp_estimator['threshold'] = self.cfg_ns.threshold
+        self.cfg_grasp_estimator['sample_based_improvement'] = 1 - int(self.cfg_ns.gradient_based_refinement)
+        self.cfg_grasp_estimator['num_refine_steps'] = 10 if self.cfg_ns.gradient_based_refinement else 20
+        print(self.cfg_grasp_estimator)  
```

Comment:
Avoid print statements if they are not necessary (I am assuming this is a leftover debug statement)